src/Analysis/Ast/Test/Microsoft.Python.Analysis.Tests.csproj

```diff
@@ -41,6 +42,11 @@
     <ProjectReference Include="..\..\Core\Impl\Microsoft.Python.Analysis.Core.csproj" />
     <ProjectReference Include="..\Impl\Microsoft.Python.Analysis.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="NSubstitute">
+      <HintPath>..\..\..\..\..\Users\mikhaila\.nuget\packages\nsubstitute\4.0.0\lib\netstandard2.0\NSubstitute.dll</HintPath>
```

Comment:
Not sure if it matters, but the hint path looks like it's specific to your system. The other `PackageReference`s don't have hints. I'm not a NuGet expert, though.